### DISCUSSION TITLE
q-select align selected option right

### DISCUSSION THREAD

**Discussion by abazanov** [quasarframework/quasar#15350] [General - Components / Directives / etc]
Hi All,

Probably very simple, but I cannot work it out for the life of me. 

![Screenshot 2023-01-31 111707](https://user-images.githubusercontent.com/5940942/215746218-d162eeda-5e6a-4bf0-a1f0-f4dd6a9a4032.png)

I need this q-select to show the selected option on the right. That is all. I have managed to align the drop-down options right by way of slots, but once a selection is made, it shows it on the left.

I am on: Pkg quasar........ v1.22.5

**Comment by david-7x0** [comment]
.q-field__native span { text-align: right; width: 100%; }
